build/evergreen_config_lib/tasks.py

```diff
@@ -604,7 +604,11 @@ def _check_allowed(self):
 
         if self.cse:
             require(self.version == 'latest' or parse_version(self.version) >= parse_version("4.2"))
-            require(self.topology == 'server')
+            if self.version == 'latest' or parse_version(self.version) >= parse_version("6.0"):
+                # FLE 2.0 Client-Side Encryption tasks on 6.0 require a non-standalone topology.
+                require(self.topology == 'server' or self.topology == 'replica_set')
```

Comment:
Fun shorthand tip, Python lets you use `in` with an iterable on the right-hand side:

```py
require(self.topology in ('server', 'replica_set'))
```

Reply:
Neat, done.